test/cctest/node_test_fixture.h

```diff
@@ -0,0 +1,85 @@
+#ifndef TEST_CCTEST_NODE_TEST_FIXTURE_H_
+#define TEST_CCTEST_NODE_TEST_FIXTURE_H_
+
+#include <stdlib.h>
+#include "gtest/gtest.h"
+#include "node.h"
+#include "env.h"
+#include "v8.h"
+#include "libplatform/libplatform.h"
+
+using node::Environment;
+using node::IsolateData;
+using node::CreateIsolateData;
+using node::CreateEnvironment;
+using node::AtExit;
+using node::RunAtExit;
+
+class ArrayBufferAllocator : public v8::ArrayBuffer::Allocator {
+ public:
+  virtual void* Allocate(size_t length) {
+    return AllocateUninitialized(length);
+  }
+
+  virtual void* AllocateUninitialized(size_t length) {
+    return calloc(length, sizeof(int));
```

Comment:
`calloc(length, 1)` - `sizeof(int)` allocates 4x more than requested.

(1 because `sizeof(char) == 1` by definition, it's the smallest addressable unit.)